server.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const server = express();
+
+server.use(express.json());
+
+const food = [
+    { id: 0, name: 'Pasta'},
+    { id: 1, name: 'Pizza'},
+    { id: 2, name: 'Soup'}
+]
+
+server.get('/food', (req, res) => {
+    res.status(200).json(food);
+});
+
+server.post('/food', (req, res) => {
+    const { name } = req.body;
+    const id = food.length;
+    food.push({ id, name });
+    res.status(201).json(food);
+});
+
+server.delete(`/food/:id`, (req, res) => {
+    const { id } = req.params;
+    if ( id ){
+        res.status(200).json({
+        message: `Food item id ${id} is deleted.`
```

Comment:
Nothing is actually being deleted here. you're using an array of objects as your database, so you could use something like`food = food.filter(e => e.id !== id )`